src/css/layout/helpers.less

```diff
@@ -10,9 +10,24 @@
   display: flex;
   align-items: center;
   justify-content: center;
-  height: 85vh;
-  width: 100vw;
+  height: 50vh;
   text-align: center;
+
+  .inline-dialog {
```

Comment:
`.centered-content` (previously used by our 404 page only) was already there. Do you think it makes sense to create a file just for this?